lib/catalog/order_state_transition.rb

```diff
@@ -20,10 +20,13 @@ def determine_order_state
 
       # TO DO: How to determine order state with pre and post order_processes?
       if item_states.include?('Failed') || item_states.include?('Denied')
+        @order.update_message("error", "Order Failed")
         'Failed'
       elsif item_states.all? { |state| state == "Completed" }
+        @order.update_message("info", "Order Completed")
         'Completed'
       elsif item_states.include?('Canceled')
+        @order.update_message("info", "Order Canceled")
```

Comment:
So, when we cancel, are we going to be basically duplicating messages due to https://github.com/RedHatInsights/catalog-api/pull/866/files#diff-327b3f8d324ce68499d4cb1c141f89af6ac2a0bdfcab1e00af12ea22b1f30fa9R47? The failed one I can see being useful because the first message would be "Order has approval errors" followed by "Order failed", but for canceled, aren't they basically the same message? Might be weird to see it twice

Reply:
Yes, removed it.